src/matter/cluster/server/OperationalCredentialsServer.ts

```diff
@@ -119,7 +119,13 @@ export const OperationalCredentialsClusterHandler: (conf: OperationalCredentials
 
                 // TODO persist fabrics
                 // TODO: depending on cases destroy the secure session and delete all data!
-
+                if (device.getFabrics().length === 0) {
+                    logger.info(`Expire all sessions because deleted fabric ${fabricIndex.index} was the last one`);
```

Comment:
Why do this?
The code below should bring to the same state since there will be no more Fabrics

Reply:
right

Reply:
PS: The specs are phrasing the case a bit different but for now you are tight... it is the same